packages/frontend/src/connectors/readonly/index.ts

```diff
@@ -1,77 +1,83 @@
-import { JsonRpcProvider } from "@ethersproject/providers";
+import { createPublicClient, createWalletClient, http } from "viem";
+import type {
+    Address,
+    Chain,
+    ConnectorData,
+    PublicClient,
+    WalletClient,
+} from "wagmi";
 import { Connector } from "wagmi";
-import { Address, Chain, ConnectorData, normalizeChainId } from "@wagmi/core";
-import { constants } from "ethers";
+import { zeroAddress } from "viem";
```

Comment:
Double `viem` import here.

Reply:
I think you meant `wagmi`. Addressed in 5d758c854076d4d0912a6d99455d4653a904c5fc